code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -363,4 +367,110 @@ static HookFunction initFunction([]()
 	{
 		*reinterpret_cast<float *>(wheelAddr + WheelXOffsetOffset) = context.GetArgument<float>(2);
 	}));
+
+	static struct : jitasm::Frontend {
+		static bool ShouldSkipRepairFunc(fwEntity* VehPointer)
+		{
+
+			for (auto it = vehicles.begin(); it != vehicles.end(); it++) {
```

Comment:
a) old iterators and not C++11 stuff?
b) std::unordered_set::find, this will be a linear scan and slow

could be replaced with `return g_skipRepairVehicles.find(vehPointer) != g_skipRepairVehicles.end();`